.github/workflows/ci-cd.yml

```
@@ -182,12 +186,17 @@ jobs:
         path: ${{ steps.pip-cache.outputs.dir }}
         restore-keys: |
             pip-ci-${{ runner.os }}-${{ matrix.pyver }}-${{ matrix.no-extensions }}-
-    - name: Update pip, wheel, setuptools, build, twine
+    - name: Update pip, wheel, setuptools, build
       run: |
-        python -m pip install -U pip wheel setuptools build twine
+        python -m pip install -U pip wheel setuptools build
     - name: Install dependencies
+      if: ${{ !endsWith(matrix.pyver, 't') }}
       run: |
         python -m pip install -r requirements/test.in -c requirements/test.txt
+    - name: Install dependencies
+      if: ${{ endsWith(matrix.pyver, 't') }}
+      run: |
+        python -m pip install -r requirements/test-ft.in -c requirements/test-ft.txt
```

No need for two steps
```suggestion
      env:
        DEPENDENCY_GROUP: >-
          test${{ endsWith(matrix.pyver, 't') && '-ft' || '' }}
      run: |
        python -Im pip install -r requirements/${DEPENDENCY_GROUP}.in -c requirements/${DEPENDENCY_GROUP}.txt
```

---

tried that, but it didn't work for normal builds https://github.com/aio-libs/aiohttp/actions/runs/17791468157/job/50569333493

---

done